src/style.css

```diff
@@ -51,6 +46,11 @@ body {
   pointer-events: none;
 }
 
```

Comment:
- [ ] New entries **do not match** the property index. The _property index_ must be set to the value of the _new array length_ (i.e. if a 5th task is added to the list, the index of that task should equal 5).

**Index mismatch in your application:**
![image](https://user-images.githubusercontent.com/12745474/135548996-588495f1-424a-417b-873f-8ee4875cc7c2.png)